ext/bert/c/decode.c

```diff
@@ -14,9 +15,12 @@
 #define ERL_BIN           109
 #define ERL_SMALL_BIGNUM  110
 #define ERL_LARGE_BIGNUM  111
+#define ERL_ENC_STRING    112
+#define ERL_UNICODE_STRING 113
```

Comment:
IIUC this is the first time we're diverging from erlang's External Term Format. And as said format does have these values (for `NEW_FUN_EXT` and `EXPORT_EXT` resp.), I think a comment explaining that we knowingly diverge here for version 2 would be useful.